deploy/kubectl/hostpath-operator.yaml

```diff
@@ -119,13 +102,17 @@ spec:
         - name: OPENEBS_IO_ENABLE_ANALYTICS
           value: "true"
         - name: OPENEBS_IO_INSTALLER_TYPE
-          value: "openebs-operator-lite"
+          value: "openebs-operator-hostpath"
         - name: OPENEBS_IO_HELPER_IMAGE
-          value: "openebs/linux-utils:2.3.0"
+          value: "openebs/linux-utils:latest"
```

Comment:
i think we should use `ci` tag here , just to be consistent with other image tag we use with github CI

Reply:
@nsathyaseelan @prateekpandey14 @kmova -- Changed linux-utils tag to `ci`.